docs/faq.md

```diff
@@ -47,4 +47,14 @@ New password for default administrator (user-xxxxx):
 
 ### I added an additional disk with partitions. Why is it not getting detected?
 
-As of Harvester v1.0.2, we no longer support adding additional partitioned disks, so be sure to delete all partitions first (e.g., using `fdisk`).
+As of Harvester v1.0.2, adding additional partitioned disks is no longer supported, be sure to delete all partitions first (e.g., using `fdisk`).
+
+### How to add ssh authorized keys to a Harvester node using a modified config file?
+
+Option 1. Non Persistent (Temporary)
+To dynamically add users in the runtime without rebooting, add ssh authorized keys to /home/rancher/.ssh/authorized_keys. These changes are not persistent between reboots.
+
+Option 2. Persistent
+After you ssh login to Harvester, add ssh authorized keys to your /oem/99_custom.yaml or 90_custom.yaml. These changes are persistent between reboots.
```

Comment:
I thought in the harvester cluster, we only have `/oem/90_custom.yaml`.
Maybe we could remove the `/oem/99_custom.yaml`?